### ISSUE TITLE
Copy to Clipboard copies with the title 

### ISSUE THREAD

**Issue by harik8** [kubernetes/website#21057]
**This is a Bug Report**

When I click on copy to clipboard it copies with the title
```
pods/resource/cpu-request-limit.yaml

apiVersion: v1
kind: Pod
metadata:
  name: cpu-demo
  namespace: cpu-example
spec:
  containers:
  - name: cpu-demo-ctr
    image: vish/stress
    resources:
      limits:
        cpu: "1"
      requests:
        cpu: "0.5"
    args:
    - -cpus
    - "2"
```

**Proposed Solution:**
It'd be nice if it's only copies the yaml content.

**Page to Update:**
https://kubernetes.io/...

**Comment by sftim** [comment]
Thanks for reporting this @barik8
/kind bug
/priority important-longterm

**Comment by prasadkatti** [comment]
The code that copies the text to the clipboard is here -
https://github.com/kubernetes/website/blob/b3c961df2f442dced554f1bd32a728a285f5a282/static/js/script.js#L20-L52

**Comment by prasadkatti** [comment]
This line -
https://github.com/kubernetes/website/blob/b3c961df2f442dced554f1bd32a728a285f5a282/static/js/script.js#L34
needs to be something like -
`target.value = document.getElementById(elem).getElementsByTagName("tbody")[0].innerText;`
in order to avoid the file name from the `<thead>` element. The `elem` here is the `<table>` element.

**Comment by tengqm** [comment]
@prasadkatti would you like to self assign this with a fix?

**Comment by kbhawkey** [comment]
A slightly different approach was taken to write the contents of the code block/element innerText.
I plan to close this issue. You can reopen if there are remaining questions.
Fixed in PR #23966 .
/close
